coderadar-test/src/test/java/io/reflectoring/coderadar/rest/analyzerconfig/DeleteAnalyzerConfigControllerIntegrationTest.java

```diff
@@ -51,7 +52,7 @@ void deleteAnalyzerConfigurationWithId() throws Exception {
   void deleteAnalyzerConfigurationReturnsErrorWhenAnalyzerNotFound() throws Exception {
     mvc()
         .perform(delete("/api/projects/0/analyzers/2"))
-        .andExpect(MockMvcResultMatchers.status().isNotFound())
+        .andExpect(status().isNotFound())
         .andExpect(
             MockMvcResultMatchers.jsonPath("errorMessage")
```

Comment:
if static imports are used for status and so on, we should also do it for jsonPath and so on (mockitot assersts, verify, ...)

Reply:
Is it really necessary to use static imports for every static method in Coderadar? If so, we should do this in an other PR, i think.